components/ContentTemplates/LinksTemplate.tsx

```diff
@@ -169,6 +169,38 @@ other places on the web */`}
 					</li>
 				</ul>
 			</TemplateSection>
+			{/* For issue #387 */}
```

Comment:
I don't think this comment is necessary

Reply:
Alright removed that! and formatted the LinksTemplate.tsx